ptf/saiacl.py

```diff
@@ -0,0 +1,6068 @@
+            port_id=self.port25,
+            type=SAI_BRIDGE_PORT_TYPE_PORT,
+            admin_state=True)
+
+        self.bridge_ports.append(port25_bp)
+
+        vlan_member2 = sai_thrift_create_vlan_member(
+            self.client,
+            vlan_id=self.vlan_oid,
+            bridge_port_id=port25_bp,
+            vlan_tagging_mode=SAI_VLAN_TAGGING_MODE_UNTAGGED)
+        self.vlan_members.append(vlan_member2)
+        self.vlan_ports.append(self.port25)
+
+        port26_bp = sai_thrift_create_bridge_port(
+            self.client,
+            bridge_id=self.default_1q_bridge,
+            port_id=self.port26,
+            type=SAI_BRIDGE_PORT_TYPE_PORT,
+            admin_state=True)
+
+        self.bridge_ports.append(port26_bp)
+
+        vlan_member3 = sai_thrift_create_vlan_member(
+            self.client,
+            vlan_id=self.vlan_oid,
+            bridge_port_id=port26_bp,
+            vlan_tagging_mode=SAI_VLAN_TAGGING_MODE_UNTAGGED)
+        self.vlan_members.append(vlan_member3)
+        self.vlan_ports.append(self.port26)
+
+        sai_thrift_set_port_attribute(
+            self.client, self.port24, port_vlan_id=vlan_id)
+        sai_thrift_set_port_attribute(
+            self.client, self.port25, port_vlan_id=vlan_id)
+        sai_thrift_set_port_attribute(
+            self.client, self.port26, port_vlan_id=vlan_id)
+
+        # Create Lag (port 27, 28) and add it to Vlan100
+        self.lag_id = sai_thrift_create_lag(self.client)
+        self.lags.append(self.lag_id)
+        lag1_bp = sai_thrift_create_bridge_port(
+            self.client,
+            bridge_id=self.default_1q_bridge,
+            port_id=self.lag_id,
+            type=SAI_BRIDGE_PORT_TYPE_PORT,
+            admin_state=True)
+        self.bridge_ports.append(lag1_bp)
+        lag_member_id1 = sai_thrift_create_lag_member(
+            self.client, lag_id=self.lag_id, port_id=self.port27)
+        self.lag_members.append(lag_member_id1)
+
+        lag_member_id2 = sai_thrift_create_lag_member(
+            self.client, lag_id=self.lag_id, port_id=self.port28)
+        self.lag_members.append(lag_member_id2)
+
+        vlan_member4 = sai_thrift_create_vlan_member(
+            self.client,
+            vlan_id=self.vlan_oid,
+            bridge_port_id=lag1_bp,
+            vlan_tagging_mode=SAI_VLAN_TAGGING_MODE_UNTAGGED)
+
+        self.vlan_members.append(vlan_member4)
+        self.vlan_ports.append(self.port27)
+        self.vlan_ports.append(self.port28)
+
+        fdb_entry = sai_thrift_fdb_entry_t(
+            switch_id=self.switch_id,
+            mac_address=self.mac,
+            bv_id=self.vlan_oid)
+        sai_thrift_create_fdb_entry(
+            self.client,
+            fdb_entry,
+            type=SAI_FDB_ENTRY_TYPE_STATIC,
+            bridge_port_id=port24_bp,
+            packet_action=mac_action)
+        self.fdbs.append(fdb_entry)
+
+    def runTest(self):
+        print("Testing AclRedirectTest")
```

Comment:
This PTF has redirect action only to port/lag. SAI also supports redirect to single nhop or ECMP. Are we adding those tests as well? If not, pls rename this test to reflect redirect to port/lag